python_modules/libraries/dagster-components/dagster_components/components/dbt_project/component.py

```diff
@@ -124,3 +128,35 @@ def _fn(context: AssetExecutionContext):
 
     def execute(self, context: AssetExecutionContext, dbt: DbtCliResource) -> Iterator:
         yield from dbt.cli(["build"], context=context).stream()
+
+
+def get_asset_key_for_model(
```

Comment:
recommend choosing a different name so you don't have to alias the one in dagster_dbt